test/e2e/scheduling/preemption.go

```diff
@@ -214,14 +214,14 @@ var _ = SIGDescribe("SchedulerPreemption [Serial]", func() {
 			if len(pods) == 0 {
 				priorityName = lowPriorityClassName
 			}
-			pods[i] = createPausePod(f, pausePodConfig{
+			pods = append(pods, createPausePod(f, pausePodConfig{
```

Comment:
How come this was passing before?

Reply:
No idea. I'm curious whether it really ran before - which looks like an obvious bug.

Reply:
This is an obvious fix, but I'm not so sure about the other one. The scheduler should work if some pods are directly assigned. Unless this is related to #88389 (the fact that we have different event handlers).

Reply:
Yup, I don't have clear clues why the other fix _can_ resolve the problem in my env. Let me dig more.